duplicates.md

```diff
@@ -0,0 +1,97 @@
+# Duplicate detection endpoint
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/submission/duplicates**
+
+Provide access to basic duplicate detection services. These services use Solr and the levenshtein distance operator
+to detect potential duplicates of a given item, useful during submission and workflow review.
+
+See `dspace/config/modules/duplicate-detection.cfg` for configuration properties and examples.
+
+## Search
+
+**GET /api/submission/duplicates/search?uuid=<:uuid>**
```

Comment:
this is not aligned to our REST architecture. We should have something like that
` /api/submission/duplicates/search/byUUID`

Reply:
yes. the angular data service will need updating too, if i am able to do this -- as i noted in the other PR comment, my understanding was that RestRepositories with searchBy methods etc require an addressable object, which the "duplicate" stubs are not (even though they are related to addressable Items).

Reply:
@kshepherd : I tried to clarify the RestRepository question in your [implementation PR here](https://github.com/DSpace/DSpace/pull/9265#discussion_r1509131702).  Please let us know if it still doesn't make sense though.